From 8acf06db501f9825c1850029724e4ce962096045 Mon Sep 17 00:00:00 2001
From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Mon, 14 Apr 2014 17:37:31 +0200
Subject: [PATCH 057/110] ARM: dts: pbab01: Enable UART1

This patch enables UART1 on the phyFLEX connector (i.MX6 uart3).

Signed-off-by: Philipp Zabel <p.zabel@pengutronix.de>
Signed-off-by: Shawn Guo <shawn.guo@freescale.com>
---
 arch/arm/boot/dts/imx6qdl-phytec-pbab01.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qdl-phytec-pbab01.dtsi b/arch/arm/boot/dts/imx6qdl-phytec-pbab01.dtsi
index c6ab762..323a647 100644
--- a/arch/arm/boot/dts/imx6qdl-phytec-pbab01.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-phytec-pbab01.dtsi
@@ -57,6 +57,10 @@
 	status = "okay";
 };
 
+&uart3 {
+	status = "okay";
+};
+
 &uart4 {
 	status = "okay";
 };
-- 
1.9.2

